### PR TITLE
delete temp files generated

### PR DESCRIPTION
**Pull request by Jiu9Shen**
## Metadata

 - Issue #

- [ ] I feel this PR is ready to be merged.
- [ ] This PR is dependent upon [PR #/ nothing]

Documentation:
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

## Testing?

- [ ] I tested on a generic Tripal Site
- [ ] I tested on a KnowPulse Clone
- [ ] This PR includes automated testing

### CODE REVIEW

File: includes/vcf_filter.trpdownload.bgzipped.inc [hunk 1]
```diff
@@ -91,6 +91,11 @@ function vcf_filter_bgzipped_generate_file($variables, $job_id = NULL, $drush_us
   print 'Combining into an archive: '.$format_tmp.'.tar'."\n";
   $command = 'tar cvf '.escapeshellarg($format_tmp).'.tar'.' -C '.escapeshellarg($bgzip_tmp_path).' '.escapeshellarg($bgzip_filename).'.gz '.escapeshellarg($bgzip_filename).'.gz.tbi';
   print shell_exec($command);
+  
+  //delete uncessary files
```
Comment (carolyncaron) [inline includes/vcf_filter.trpdownload.bgzipped.inc]: I would suggest moving this after you check your output exists. This way, if an error is returned, these temporary files might be used for debugging.

File: includes/vcf_filter.trpdownload.qual_matrix.inc [hunk 2]
```diff
@@ -77,6 +77,11 @@ function vcf_filter_qual_matrix_generate_file($variables, $job_id = NULL, $drush
   print str_repeat('-', 60) . "\n";
   $command = 'vcftools --vcf ' . escapeshellarg($filtered_sorted_vcf) . ' --geno-depth --out ' . escapeshellarg($format_tmp);
   print shell_exec($command);
+  //delete filterd sorted vcf file
```
Comment (carolyncaron) [inline includes/vcf_filter.trpdownload.qual_matrix.inc]: See my other comment; try moving this after the next if block

Comment (Jiu9Shen) [reply includes/vcf_filter.trpdownload.qual_matrix.inc]: Changes are made, could you check the code and merge this pr if it works.

**Comment by carolyncaron** [comment]
Tested and merged!

**Pull request merged**